pkg/engine/engine_test.go

```diff
@@ -187,7 +187,23 @@ func TestParallelRenderInternals(t *testing.T) {
 	wg.Wait()
 }
 
-func TestRenderErrors(t *testing.T) {
+func TestParseErrors(t *testing.T) {
+	vals := chartutil.Values{"Values": map[string]interface{}{}}
+
+	tplsUndefinedFunction := map[string]renderable{
+		"undefined_function": {tpl: `{{foo}}`, vals: vals},
+	}
+	_, err := new(Engine).render(tplsUndefinedFunction)
+	if err == nil {
+		t.Fatalf("Expected failures while rendering: %s", err)
```

Comment:
`err` will always be nil here, so I would just use `t.Fatal`

Reply:
I'm not sure I understand. If `err` is nil, that means the `render` function didn't work as expected.

Reply:
This was just noting that you don't need to use `Fatalf` and pass an `err` because it doesn't provide any additional info. e.g. `t.Fatal("Expected failures while rendering")`. But it is just a nit anyway 🙂